Examples/AferoLab/AferoLab/Base.lproj/LaunchScreen.storyboard

```diff
@@ -22,7 +20,7 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <imageView userInteractionEnabled="NO" contentMode="center" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="Startup" translatesAutoresizingMaskIntoConstraints="NO" id="VLC-C0-pOK">
-                                <rect key="frame" x="87" y="210" width="200" height="246"/>
+                                <rect key="frame" x="87.5" y="210.5" width="200" height="246"/>
```

Comment:
No big whup, but did something get nudged? Looks like something was knocked out of alignment.